app/helpers/application_helper.rb

```diff
@@ -54,4 +54,8 @@ def enable_google_analytics(cookies)
       cookies[CookiesController::COOKIE_CHOICE_NAME] == "accepted" &&
       !cookies[CookiesController::NO_ANALYTICS_MODE]
   end
+
+  def beta_pilot_url?
+    request.base_url.include?("cloud-platform")
```

Comment:
This isn't an ideal test, because it's almost impossible to replicate in development and test environments. It's also reliant on something that happens to be the case (the environments we want to show this on _happen to_ have "cloud-platform" in the URL and the environments we don't _happen_ not to), but it would be better to rely on something that's a bit more definite. I suggest that we want to show this at times where the following applies:

- We have defined a "primary" host for this environment
- This environment is _not_ the primary host.

If you look in ApplicationController#redirect_to_primary_host you'll see a similar check defined. I think we could copy that here, so this method would become `def using_non_primary_url?` or something like that.